packages/terra-paginator/src/ControlledProgressivePaginator.jsx

```diff
@@ -192,7 +192,7 @@ class ControlledProgressivePaginator extends React.Component {
     );
 
     const navigationMessage = (
-      <VisuallyHiddenText aria-live="polite" aria-relevant="additions text" text={this.state.selectedPageMessage} />
+      <VisuallyHiddenText aria-live="polite" aria-relevant="additions text" text={pageLabel ? pageDetails : this.state.selectedPageMessage} />
```

Comment:
we need to include string `is selected` (translated string) as well along with pageDetails

Reply:
> we need to include string `is selected` (translated string) as well along with pageDetails

Added 'is selected' string.